### DISCUSSION TITLE
Data Fetching with tanstack query

### DISCUSSION THREAD

**Discussion by amirzahavi** [solidjs/solid-router#278] [Q&A]
Is there a way to use `createQuery` inside of the data fetching function (without `createReousrce`)? 
I've tried it in my project and it didn't seem to work, also tried to set `suspense: true`.

And if possible, does it support the `refetchInterval` functionality of the Tanstack query?

**Comment by ryansolid** [comment]
createQuery uses createResource under the hood so I don't see why it wouldn't work. Might have more luck asking over there.
